sql/moz-fx-data-marketing-prod/ga_derived/mobile_installs_adjust_v1/query.sql

```diff
@@ -0,0 +1,63 @@
+SELECT
+  date,
+  app,
```

Comment:
we are already filtering for Firefox only on the WHERE, so I don't think we need this since it'll all have the same value.